src/export.ts

```diff
@@ -30,6 +25,42 @@ export async function generateVCard(c: Contact) {
   // https://datatracker.ietf.org/doc/html/rfc6350#section-3.3
   vcf += `X-CALLSIGN:${c.callsign}\n`;
 
+  vcf += `X-STATION-TYPE:${c.cardType}\n`;
+
+  // this is hopefully not that wrong
+  if (c.cardType == "repeater") {
+    
+    // using `X-LOCATION` instead of `ADR` or `GEO` since the inputted location could be something like "UI Steam Plant, Moscow, ID" - which isn't a mailing address, and GEO wants lat/long
+    vcf += `X-LOCATION:${c.location}\n`;
+
+    vcf += `X-REPEATER-INFO:`
+    
+    if (c.frequency) {
+      vcf += `${c.frequency}`;
+    }
+    vcf += `;`
+    if (c.offset) {
+      vcf += `${c.offset}`;
+    }
+    vcf += `;`
+    if (c.tone) {
+      vcf += `${c.tone}`;
+    }
+    vcf += `;`
+    if (c.rxtone) {
+      vcf += `${c.rxtone}`
+    }
+    vcf += `;\n`
```

Comment:
could you add a comment or something documenting this format you've come up with? it seems like someone looking at the raw .vcf for whatever reason would have a hard time figuring out what's going on